src/dot_recognize/src/point_number_publisher.py

```diff
@@ -135,10 +135,10 @@ def numbercallback(self,data):
 
 if __name__ == '__main__':
     bridge=CvBridge()
-    cap =cv2.VideoCapture(2)
+    cap =cv2.VideoCapture(0)
     bridge=CvBridge()
     ret, frame=cap.read()
     webcam = bridge.cv2_to_imgmsg(frame,"bgr8")
     rospy.init_node('point_number_pub')
     s=rospy.Service('dot_recognize',Int16,numbercallback)
```

Comment:
Service does not support `std_msgs.msg.Int16`. I see you declared a service type, you should use it.